frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -36,11 +32,23 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
 
   const basePath = resourcePath(getVMLikeModel(vmiLike).kind, name, namespace);
   const DisksTitle = React.useCallback(
```

Comment:
could we add also LUN's to not disregard anything? Floppies are not necessary as they will be deprecated soon.

Reply:
added LUN, hidden if count < 1 :+1: